webhook-runners/src/main.rs

```diff
@@ -15,6 +17,18 @@ async fn main() -> Result<(), Error> {
         .with_max_level(tracing::Level::INFO)
         .init();
 
+    let app_id = std::env::var("GITHUB_APP_ID").unwrap();
+    let private_key = std::env::var("GITHUB_APP_PRIVATE_KEY").unwrap();
+    octocrab::initialise(
+        Octocrab::builder()
+            .app(
+                AppId::from(app_id.parse::<u64>().unwrap()),
+                EncodingKey::from_rsa_pem(private_key.as_bytes()).unwrap(),
```

Comment:
cant it be created from a string?

Reply:
I don't see that option here: https://docs.rs/jsonwebtoken/latest/jsonwebtoken/struct.EncodingKey.html#method.from_rsa_pem